array_methods_lab.js

```diff
@@ -0,0 +1,185 @@
+// console.log('-----QUESTION 1------')
+// /*iven an array of numbers, write a function that
+// returns a new array where every element is incremented by 10.*/
+//
+// let numArr = [1,2,3,4,5]
+// let addTen = numArr.map((crrElem) => {
+//   return crrElem +  10
+// })
+// console.log(addTen)
+
+
+
+
+// console.log('********QUESTION 2*********')
+// /*Given an array, return a new array where every
+// non-string element is converted to an empty string.*/
+//
+// let mixedArr = ["a", 123, "b", "c", {name: "cat"}]
+// let strArr = mixedArr.map((crrElem) => {
+//   if (typeof(mixedArr) === 'number') {
+//   return ""
+//   }
+//   return mixedArr
+// })
+// console.log(strArr)
+
```

Comment:
So within your map method, you redefined each element in the array as crrElem ... bc of that you should be comparing the typeof crrElem ... because right now you comparing an array to a 'number' type. 

What is your reasoning for the comparison to a number ?  We need to be sure of exactly what you are looking for and what you need to accomplish with this function; before we even start coding. If the element is not a string then we want to do something, so we should be comparing the typeof to a string. 

Let me know if you would like to go over this question with me.